request/post_process.go

```diff
@@ -0,0 +1,39 @@
+package request
+
+import (
+	"io/ioutil"
+
+	"github.com/robertkrimen/otto"
+)
+
+// PostProcess processes the executed requests using the post processing script
+func PostProcess(options *ExecutionOptions, executedRequests []ExecutedRequestResponse, responseErr error) error {
```

Comment:
@visola 👋 

Hope you're doing well - I've just been doing some lurking 😅 

I haven't read or written `go` in a while so I apologize if I'm missing something - is `responseErr` unused?

Is it a placeholder for some future implementation?

Reply:
Yes, this was just the initial work for #77. Still a lot of work to be done.
I'm factoring most of the code to simplify how all of this works. There's a branch of you're interested.